stage4.1_first_application/pycasa/model/image_file.py

```diff
@@ -0,0 +1,20 @@
+import PIL.Image
+from PIL.ExifTags import TAGS
+
+from traits.api import Dict, File, HasStrictTraits, observe, Property
+
+
+class ImageFile(HasStrictTraits):
+    """ Model to hold an image file.
+    """
+    filepath = File
+
+    metadata = Dict
```

Comment:
Alternatively, if we want to talk about `Property` traits, we could do `metadata = Property(depends_on="filepath"`, with a `cached_property` on the getter.

Reply:
Love it. Done.